app/scripts/modules/ecs/src/serverGroup/configure/wizard/container/Container.tsx

```diff
@@ -59,7 +61,18 @@ export class Container extends React.Component<IContainerProps, IContainerState>
       dockerImages: cmd.backingData && cmd.backingData.filtered ? cmd.backingData.filtered.images : [],
       targetGroupMappings: cmd.targetGroupMappings,
       targetGroupsAvailable: cmd.backingData && cmd.backingData.filtered ? cmd.backingData.filtered.targetGroups : [],
+      containerMappings: cmd.containerMappings,
     };
+
+    if (!cmd.useTaskDefinitionArtifact) {
```

Comment:
You don't need to check useTaskDefinitionArtifact since Container.tsx is only invoked when it's using containerInputs (and useTaskDefArtifact is false).